hypertrace-ingester/build.gradle.kts

```diff
@@ -30,11 +30,19 @@ dependencies {
   implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.8")
   implementation("org.hypertrace.core.datamodel:data-model:0.1.9")
   implementation("org.hypertrace.core.viewgenerator:view-generator-framework:0.1.14")
+  implementation("com.typesafe:config:1.4.0")
```

Comment:
should this have been testImplementation? I don't see any main classes referencing this, and usually adding a version anchors in some way